experimental/packages/otlp-exporter-base/src/types.ts

```diff
@@ -52,4 +52,5 @@ export interface OTLPExporterConfigBase {
   /** Maximum time the OTLP exporter will wait for each batch export.
    * The default value is 10000ms. */
   timeoutMillis?: number;
+  alwaysUseXhr?: boolean;
```

Comment:
rather than expose a new API property called `always` (when it can't), I would prefer this to be an array of "transport" preferences...

As you may not be able to use XHR and you might want to use fetch instead (like in a worker).

Suggestion: Something like what we do in [Application Insights](https://github.com/microsoft/ApplicationInsights-JS/blob/2a1c332f7039eb3e13711a0fa3732541c562b58f/channels/1ds-post-js/src/DataModels.ts#L153-L164)
With the "available" transports [here](https://github.com/microsoft/ApplicationInsights-JS/blob/2a1c332f7039eb3e13711a0fa3732541c562b58f/shared/AppInsightsCore/src/JavaScriptSDK.Enums/SendRequestReason.ts#L58-L75)

This allows the user to "request" their preferred transport (sendBeacon, XHR, fetch) and the SDK will use the most appropriate transport for the given scenario. The difference between the `transports` and `unloadTransports` that we have is to allow the user to have a different preference once we detect that the page is navigating away (we listen only to the events)